tests/Unit/Service/Authenticator/Exception/InvalidAuthenticatorTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+namespace Unit\Service\Authenticator\Exception;
+
+use Shopgate\CloudIntegrationSdk\Service\Authenticator\Exception\InvalidAuthenticator as InvalidAuthenticatorException;
```

Comment:
there is no need to make an alias for this, but if you really want the user to know it's an exception, you can use a better convention of using something like this:
```
use Shopgate\CloudIntegrationSdk\Service\Authenticator\Exception;
...
... = new Exception\InvalidAuthenticator();
```
This allows for adding other exceptions later form that namespace, plus not needing to create extra aliases that need looking up, plus already has that Exception marker that you wanted.